ihatemoney/templates/list_bills.html

```diff
@@ -128,6 +128,9 @@ <h3 class="modal-title">{{ _('Add a bill') }}</h3>
                 <td class="bill-actions">
                     <a class="edit" href="{{ url_for(".edit_bill", bill_id=bill.id) }}" title="{{ _("edit") }}">{{ _('edit') }}</a>
                     <a class="delete" href="{{ url_for(".delete_bill", bill_id=bill.id) }}" title="{{ _("delete") }}">{{ _('delete') }}</a>
+                    {% if bill.document %}
```

Comment:
i know i kinda match the lines 👆but title exactly like the next note seems useless...
https://inclusive-components.design/tooltips-toggletips/
i would use some documentation in place of this...

Reply:
Title is useful on mouse over events to display a small text box. I don't want to refactor this for now but according to your link we can use tooltips in future.